### PR TITLE
Update FieldType.cs

### LINKED ISSUE #504: storeName is missing in the FieldNameEnum
Hi everybody,

will implementing the adyen API I noticed that the storeName is missing in the FieldNameEnum. You can reproduce this Bug when you try to add a store and the store name contains a invalid char like the @ symbol. When you try do deserialize the response with the FieldNameEnum you will get a exception.

Thanks for your help

Error converting value "storeName" to type 'System.Nullable`1[Adyen.Model.MarketPay.FieldType+FieldNameEnum]'. Path 'invalidFields[0].fieldType.fieldName'

### PR DESCRIPTION
**Pull request by AlexandrosMor**
## Summary

Adding missing enums
## Tested scenarios

**Fixed issue**:  
fixes #504

### CODE REVIEW

File: Adyen/Model/MarketPay/FieldType.cs [hunk 1]
```diff
@@ -601,9 +1004,9 @@ public override string ToString()
         {
             var sb = new StringBuilder();
             sb.Append("class FieldType {\n");
-            sb.Append("  Field: ").Append(Field).Append("\n");
-            sb.Append("  FieldName: ").Append(FieldName).Append("\n");
-            sb.Append("  ShareholderCode: ").Append(ShareholderCode).Append("\n");
+            sb.Append("  field: ").Append(Field).Append("\n");
+            sb.Append("  fieldName: ").Append(FieldName).Append("\n");
```
Comment (AlexandrosMor) [inline Adyen/Model/MarketPay/FieldType.cs:1008]:
```suggestion
            sb.Append("  FieldName: ").Append(FieldName).Append("\n");
```

File: Adyen/Model/MarketPay/FieldType.cs [hunk 2]
```diff
@@ -601,9 +1004,9 @@ public override string ToString()
...
+            sb.Append("  shareholderCode: ").Append(ShareholderCode).Append("\n");
```
Comment (AlexandrosMor) [inline Adyen/Model/MarketPay/FieldType.cs:1009]:
```suggestion
            sb.Append("  ShareholderCode: ").Append(ShareholderCode).Append("\n");
```

File: Adyen/Model/MarketPay/FieldType.cs [hunk 3]
```diff
@@ -1006,7 +1006,7 @@ public override string ToString()
             sb.Append("class FieldType {\n");
             sb.Append("  field: ").Append(Field).Append("\n");
```
Comment (wboereboom) [inline Adyen/Model/MarketPay/FieldType.cs]: shouldn't field and fieldname also be capitalized?

**Review by zaiddreakh — APPROVED** [review]

**Review by wboereboom — APPROVED** [review]
Hard to review for correctness. Have we tested these new types?

File: Adyen/Model/MarketPay/FieldType.cs [hunk 4]
```diff
@@ -48,517 +48,920 @@ public enum FieldNameEnum
             /// </summary>
             [EnumMember(Value = "accountCode")]
```
Comment (zaiddreakh) [inline Adyen/Model/MarketPay/FieldType.cs:49]: why do all enumMember values in this file start with lowercase? in other files they start with uppercase

**Pull request merged**